Source/WebCore/style/StyleTreeResolver.h

```diff
@@ -116,6 +116,12 @@ class TreeResolver {
         bool needsUpdateQueryContainerDependentStyle { false };
         IsInDisplayNoneTree isInDisplayNoneTree { IsInDisplayNoneTree::No };
 
+#if ENABLE(ACCESSIBILITY_ISOLATED_TREE)
+        // Used to optimize accessibility isolated tree updates.
+        bool isAXFontChangeSubtree { false };
+        bool isAXColorChangeSubtree { false };
+#endif
```

Comment:
This should go at the bottom of the structure definition, near all the other member variables.

Can you explain the name of this variable to me? I don't really understand, even after looking at how it's used. In any case, having `AX` as part of the name seems inaccurate. AX / accessibility is what consumes the bools, but not descriptive of the actual state it represents.